### PR TITLE
Create user model

### PR DESCRIPTION
**Pull request by Nitishrestha**
create user model with properties.

### CODE REVIEW

File: src/main/java/com/kat/bachaat/model/User.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    private String lastName;
+    private String email;
+    private String address;
+    private int activationCode;
+    private Long mobileNumber;
+    private boolean active = false;
+
+    public User() {
+        this.activationCode = new Random().nextInt(900000) + 100000;
+    }
```
Comment (kripesh123) [inline src/main/java/com/kat/bachaat/model/User.java]: Generating activation code is not encourage here

File: src/main/java/com/kat/bachaat/model/User.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+public class User {
+    private Long userId;
+    private String firstName;
+    private String middleName;
+    private String lastName;
+    private String email;
+    private String address;
+    private int activationCode;
+    private Long mobileNumber;
```
Comment (kripesh123) [inline src/main/java/com/kat/bachaat/model/User.java:13]: Add one to many user role.

File: src/main/java/com/kat/bachaat/model/User.java (hunk 1 above)
Comment (Nitishrestha) [reply src/main/java/com/kat/bachaat/model/User.java]: Okay, I will remove from here and place it somewhere.

File: src/main/java/com/kat/bachaat/model/User.java (hunk 2 above)
Comment (Nitishrestha) [reply src/main/java/com/kat/bachaat/model/User.java:13]: Okay.

**Pull request merged**